interfaces/HO-Portal/src/app/program/edit-person-affected-popup/edit-person-affected-popup.component.html

```diff
@@ -58,6 +58,13 @@
         [inProgress]="inProgress?.whatsappPhoneNumber || false"
         class="ion-margin-vertical"
       ></app-update-property-item>
+      <app-update-property-item
```

Comment:
So I'd suggest to put the whole FSP-specific block here.
That also reduces the handling of the `fspList`-object a bit.